pgcli/packages/pgspecial/iocommands.py

```diff
@@ -68,6 +68,30 @@ def open_external_editor(filename=None, sql=''):
 
     return (query, message)
 
+@special_command('\\i', '\\i file', 'List or execute named queries.')
+def execute_from_file(cur, pattern, **_):
+    if pattern != '':
```

Comment:
In python empty strings are falsy. So you can change this `if` condition as follows:

```
if pattern:
```